hudi-client/src/main/java/org/apache/hudi/index/HoodieIndex.java

```diff
@@ -43,6 +44,8 @@
  */
 public abstract class HoodieIndex<T extends HoodieRecordPayload> implements Serializable {
 
+  public static EnumSet<IndexType> GLOBAL_INDICES = EnumSet.of(IndexType.GLOBAL_BLOOM, IndexType.HBASE);
```

Comment:
This is not very desirable since this is only required for deletion purposes and does not add any other value. Each implementation of the Index already has a method that describes whether it's global or not.